libs/shared/barista-definitions/src/lib/barista-definitions.ts

```diff
@@ -155,3 +156,13 @@ export type BaContentTypes =
   | BaIndexPageContent
   | BaIconOverviewPageContent
   | BaCategoryNavigationContent;
+
+export interface TOC {
+  headlines: Headline[];
+}
```

Comment:
Does this need its own interface, it is basically `Headline[]`. I don't think it needs to be nested.